justfile

```diff
@@ -59,7 +59,13 @@ install-denv:
 
 # configure how ldmx-sw will be built
 configure *CONFIG:
-    denv cmake -B build -S . {{ CONFIG }}
+    denv cmake -B build -S .  -DADDITIONAL_WARNINGS=ON  -DENABLE_CLANG_TIDY=ON  {{ CONFIG }}
+
+configure-force-error *CONFIG:
+    just  configure -DWARNINGS_AS_ERRORS=ON {{ CONFIG }}
```

Comment:
```suggestion
configure-force-error *CONFIG: (configure -DWARNINGS_AS_ERRORS=ON CONFIG)
```

Reply:
Implemented it as `configure-force-error *CONFIG: (configure "-DWARNINGS_AS_ERRORS=ON" CONFIG)`

Reply:
We found out that `just` treats `-` as tokens within its own syntax and so we need to wrap the `-DWARNINGS_AS_ERRORS=ON` with `"` to inform `just` that it should be treated as an argument to the recipe and not apart of its own syntax.